flow/core/kernel/vehicle/base.py

```diff
@@ -131,6 +131,23 @@ def remove(self, veh_id):
     def apply_acceleration(self, veh_id, acc):
         """Apply the acceleration requested by a vehicle in the simulator.
 
+        In SUMO, this function applies slowDown method which applies smoothing.
+
+        Parameters
+        ----------
+        veh_id : str or list of str
+            list of vehicle identifiers
+        acc : float or array_like
+            requested accelerations from the vehicles
+        """
+        raise NotImplementedError
+
+    def apply_acceleration_not_smooth(self, veh_id, acc):
```

Comment:
is this being used in this PR? I can find a function call for it. If it's not, let's leave it for another PR

Reply:
you can delete this function. I added this when we were testing slowDown vs setSpeed methods. I think we don't need this anymore.